bootstrapper/internal/initserver/initserver.go

```diff
@@ -161,9 +161,15 @@ func (s *Server) Init(ctx context.Context, req *initproto.InitRequest) (*initpro
 	// Check if we are running on a CVM
 	_, isCVM := s.issuer.(*snp.Issuer)
 
+	clusterName := req.ClusterName
```

Comment:
Should we add a `TODO` or similar to keep track of when we can depend on `req.ClusterName` to be set?

Reply:
Since these changes only affect init, we can probably depend on the value being set as soon as the PR is merged.
We might want to do other checks or leave this check in as is if we want to ever offer the bootstrappers gRPC endpoints as a public interface.

Reply:
I thought that this was to eventually allow skew between released the CLI and Bootstrapper versions. 
But I agree that we should either decide that we only do validation in the config, or perform even more checks here. The discussion behind this is if we want to enforce that during create and init CLI version == VM image version == microservice version holds. 

I think that even the discussion if we want to expose the init endpoint as public interface is very far in the future. Right now, I think that the negative side outweighs the positive by a good margin.